tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,309 @@
+    server_name _;
+    location / {
+        proxy_pass  http://127.0.0.1:7700;
+    }
+}
+EOF
+```
+
+finally we need to enable and start the nginx service as we did before, to make sure it is always available
+
+```bash
+# Reload the operating system daemons / services
+$ systemctl daemon-reload
+
+# Enable and start nginx service
+$ systemctl enable nginx
+$ systemctl start nginx
+```
+
+Our MeiliSearch is now up, deployed in a production environment, using a safe API key, and being served by a Reverse Proxy Nginx. We should now be able to do requests to our server from the outside world by doing any HTTP request MeiliSearch accepts at our own IP address (http://<your-ip-address>). The IP address is the same you used to connect to your machine via SSH in Step 1.
+
+
+> If you want to learn more about using Nginx as a Reverse Proxy, see its dedicated documentation [Nginx as a Reverse Proxy](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)
+
+Our only problem now is that we are using MeiliSearch under HTTP, without any additional security. This is an important issue, as the content that is being passed via HTTP can easily be read or modified by attackers, and someone could get full or partial acces to your data. In order to prevent this, it's important to use the SSL protocol, which will let us use HTTPS, a verification and encrypted way of transmitting data.
+
+### 3.2. Activating SSL (HTTPS) on your MeiliSearch
+
+SSL will let the user or client establish an authenticated connection to your MeiliSearch. In this way, a user can verify MeiliSearch's identity before sending sensitive data or making any request. After the identity is verified, data is sent in an encrypted way that only MeiliSearch server will be able to decrypt, giving you a fast, reliable and automatic layer of security.
+
+In order to activate SSL, you want to use your own domain name. The first step is to register your own domain name, and change the DNS records. To make your domain name point to your newly installed MeiliSearch server, you just need to add a `A record` to the IP address that you used to connect to the server. This process is simple, but varies for every domain name provider, so we will not cover that process in this artcile.
+
+>When you register a domain name, and add an `A record`, you should be automaticaly be able to request MeiliSearch directly by using that domain name. It means, for illustration purposes, that if you registered your domain name `example.com`, requests for your MeiliSearch indexes should be done at http://example.com/indexes
+
+Now that your domain name is setup, you are ready to set SSL and use HTTPS. We will present two different options to accomplish this goal. First of all, you can use [Certbot](https://certbot.eff.org/), an amazing, free and very easy to use tool. Finally we will see what you need to do if you already have some SSL certificates issued from a `Certificate Authority or CA` for your domain name.
```

Comment:
The sentence announces two options, which is great. Then starts with presenting the first one that will be covered and also what will be covered after the two options. This is a little weird, maybe make a quick presentation of both options in this paragraph.

Reply:
I tried to change that in the new commit, thanks!